### PR TITLE
Update for dask-gateway 0.9.0

### PR DESCRIPTION
**Pull request by TomAugspurger**
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by TomAugspurger** [comment]
@conda-forge-admin, please rerender

**Comment by TomAugspurger** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 package:
   version: {{ version }}
   name: {{ name|lower }}
 
 source:
   url: https://github.com/dask/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: b7dd60b37953b804bebb999f97263cd0f3493d985f9d922d5e89ee52edf17234
+  sha256: de99927a7f2e97f53b4c0ab7a784fc36ccf5dba22fb1cb15c5bd7d29e5fe1990
+
```
Comment (jcrist) [inline recipe/meta.yaml:11]: Github won't let me comment on lines outside the diff. You'll need to delete the `entry_points` below, these are gone in this release.

**Review by jcrist — COMMENTED** [review]
Thanks Tom! Was hoping the conda-forge bots would get to it, but things seem to be slow.

**Comment by TomAugspurger** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (jcrist) [reply recipe/meta.yaml:11]: Oop, there's also some CLI tests below as well that should be deleted.

Comment (TomAugspurger) [reply recipe/meta.yaml:11]: Whoops, I even made a mental note to remove those, but got forgot about it when checking if dask-gateway-server was still a command.

**Review by jcrist — APPROVED** [review]

**Comment by jcrist** [comment]
Thanks Tom!

**Pull request merged**